api/src/controllers/db-config.js

```diff
@@ -0,0 +1,16 @@
+const auth = require('../auth');
+const serverUtils = require('../server-utils');
+const dbConfigService = require('../services/db-config');
+
+module.exports = {
+  getAttachments: (req, res) => {
+    auth.getUserCtx(req)
+      .then(async userCtx => {
```

Comment:
I'm not sure why you're using async in a promise chain. 
Please use standard promises instead.